portfolio/src/main/webapp/script.js

```diff
@@ -35,9 +35,41 @@ async function helloName() {
   });
 }
 
+/**fetches comments from the server and adds them to the DOM*/
+async function loadComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments);
+    const commentsListElement = document.getElementById('comments-list');
+    comments.forEach((comment) => {
+      commentsListElement.appendChild(createCommentElement(comment));
+    })
+  });
+}
+
 /** Creates an <li> element containing text. */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+function createCommentElement(comment) {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comment';
+
+  console.log(comment);
```

Comment:
As above.  I'll stop calling them out specifically now - I think you get the idea. :)